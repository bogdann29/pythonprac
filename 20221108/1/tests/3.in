a = DivStr("ABFGJKLMNRSTUWXYZ")
print(* a // 4)
print(a % 3)
print(* a % 10 // 2)
print(a.lower() % 5)
print(* a[1:6] // 4)
print(DivStr())
