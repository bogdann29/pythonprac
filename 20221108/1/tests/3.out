ABFG JKLM NRST UWXY
YZ
STU WXY
yz
B F G J

